cmd/validate.go

```diff
@@ -62,7 +62,7 @@ func (vc *validateCmd) run(c *cobra.Command) error {
 
 	var manifestFiles []safeguards.ManifestFile
```

Comment:
you should extract this whole block into just `manifestFiles, err := safeguards.GetManifestFiles`. You already have the logic for detecting if it's a directory in preprocessing.go

Reply:
I left the blocks as-is to prevent any breaking changes and so that we wouldn't be exposing the helm/kustomize functionality to users just yet. I can make the change and just add more tests in the next PR for the run() function